src/mscorlib/src/Microsoft/Win32/Win32Native.cs

```diff
@@ -916,12 +916,12 @@ internal static bool DoesWin32MethodExist(String moduleName, String methodName)
         // a web client asking a server for "http://server/com1.aspx" and
         // then causing a worker process to hang.
         [System.Security.SecurityCritical]  // auto-generated
-        internal static SafeFileHandle SafeCreateFile(String lpFileName,
+        internal static Win32NativeCloseHandle SafeCreateFile(String lpFileName,
```

Comment:
I'm just concerned that either (1) we miss something or (2) we end up needing to re-enable these as we continue our work. I'd rather "fix" them unless you feel strongly about it.

One additional thing I could do is use `SafeHandle` for the definitions instead (on APIs that take `SafeFileHandle` arguments). Or `SafeHandleZeroOrMinusOneIsInvalid`?